fecgov/openFEC
# remove unused python libraries

## Summary (required)

- Resolves #3955 

_Include a summary of proposed changes._

1. Removed one unused python packages from requirements.txt.
   - prompt-toolkit 
2. Removed the unused python packages from requirements-dev.txt.
   - ipython 
   - faker 
   - pygments
   - pandas 
   - xlrd
3. pandas package was mainly used  to extract and load data into few table by running the python scripts in `manage.py` via command line. We can automate the data loads in a migration file and run across all our environments. Removed the python scripts that load  data in manage.py. Also deleted the unused dump files from the repo.

## How to test the changes locally

- checkout branch `feature/3955-remove-unused-pkgs` 
- run `pip install -r requirements.txt` 
- run `pip install -r requirements-dev.txt` 
- run `pytest` (all tests should pass)
- export `SQLA_CONN` to point to dev/ stage or prod db
- start server
- test API endpoints. Make sure there are no errors in the terminal.
- test CMS-API interaction.
